extensions/transaction/transaction.fragment.yaml

```diff
@@ -30,16 +31,15 @@ paths:
                 type: string
               description: A string to ensure the item has not been modified
           content:
-            application/geo+json:
-              schema:
-                type: string
-            text/html:
+            application/json:
               schema:
-                type: string
+                $ref: '#/components/responses/Feature' 
```

Comment:
This is causing an error in Circle:
```
167:23    error  invalid-ref                    '#/components/responses/Feature' does not exist
```

because Feature is not defined in the components->responses section (around line 228)

Reply:
This has been introduced with #11 - meabe we should actually merge OAFeat.yaml and STAC.yaml? Or fix the merge tool?

Reply:
I'm fine with either route. Though I'm not sure how to fix the merge tool. And not sure exactly how to merge them by hand, do you have to line up the sections? Or just put them both in one?

Reply:
I think manual merge is probably the fastest and easiest, not worth messing with the merge tool for a one-time merge that isn't going to change much between versions. 
We'd have to combine like sections.

Reply:
Ok. If you can take a crack at that then go for it Matt. If not I'll take it after I do compliance/item collection / relative links

Reply:
I'm not sure it's a good idea to just merge them as they are. I think we should try to already split them according to the conformance classes and leave the OAF schema as untouched as possible to help with future migrations. Still, it should somehow reflect STAC of course.

Reply:
I'm leaning towards just reverting this change, and go back to a string and just not have it fully aligned. Or could also just leave this PR out entirely of beta.1 - just put in a note that they are very similar and we are working to fully align.

Reply:
It was easier than expected, changed a ref and that was basically it. Not sure everything is 100% correct, but should be better than before...